CollapseLauncher/Classes/GameManagement/GameSettings/Genshin/RegistryClass/GeneralData.cs

```diff
@@ -24,40 +24,77 @@ internal class GeneralData
         //Using guide from https://github.com/Myp3a/GenshinConfigurator/wiki/Config-format
         //Thanks Myp3a!
 
-        public int mtrAbortTimeOutCount { get; set; } = 3;
-
         /// <summary>
-        /// Could be for PlayStation multiplayer stuff
+        /// deviceUUID<br/>
+        /// This is supposed to be empty
         /// </summary>
-        public bool onlyPlayWithPSPlayer { get; set; } = false;
-
-        public int urlCheckErrorTraceCount { get; set; } = 30;
-
-        public decimal uiPaperWhite { get; set; } = 0.0m;
-        public decimal scenePaperWhite { get; set; } = 0.0m;
```

Comment:
Are we removing it because we no longer need to track them? They're still defined as of 4.0.0, so we should probably either comment them out to use for future reference or store the information somewhere.

Reply:
Wasn't removed, its moved to somewhere else, following the official sorting of the General Data JSON

Reply:
Ah yea, saw it at the end of the file, makes sense. Thanks for clarifying.